js/main.js

```diff
@@ -0,0 +1,64 @@
+
+'use strict'
+
+{
+
+  const contents = document.getElementById('text');
+  const addButton = document.getElementById('add');
+  const taskValue = document.getElementById('text');
+  const table = document.getElementById('table');
+  console.log(table);
+  const tbody = document.createElement('tbody');
+
+  table.appendChild(tbody);
+
+  const items = [];
+
+  const showArray = items => {
```

Comment:
確かに `配列を表示` するんですが…名前が実装に寄りすぎている気がします。
例えば `showTasks` 等はいかがでしょうか？
```suggestion
  const showTasks = items => {
```